src/main/scala/com/lucidworks/spark/example/ml/SVMExampleScala.scala

```diff
@@ -0,0 +1,161 @@
+package com.lucidworks.spark.example.ml
+
+import java.util
+
+import com.lucidworks.spark.SparkApp
+import com.lucidworks.spark.analysis.LuceneTextAnalyzer
+import org.apache.commons.cli.{CommandLine, Option}
+import org.apache.spark.mllib.classification.{SVMWithSGD, SVMModel}
+import org.apache.spark.mllib.evaluation.BinaryClassificationMetrics
+import org.apache.spark.sql.{SaveMode, SQLContext}
+import org.apache.spark.sql.types.{StringType, StructField, StructType}
+import org.apache.spark.storage.StorageLevel
+import org.apache.spark.{SparkContext, SparkConf}
+import org.apache.spark.mllib.regression.LabeledPoint
+import org.apache.spark.sql.Row
+import org.apache.spark.api.java.function.Function
+import org.apache.spark.mllib.feature.HashingTF
+import org.apache.spark.mllib.feature.Normalizer
+import org.apache.spark.mllib.feature.StandardScaler
+import org.apache.spark.mllib.feature.StandardScalerModel
+import scala.collection.JavaConverters._
+import scala.collection
+
+import scala.collection.immutable
+
+object SVMExampleScala {
+  val DEFAULT_NUM_FEATURES = "1000000"
+  val DEFAULT_NUM_ITERATIONS = "200"
+  val DefaultZkHost = "localhost:9983"
+  val DefaultCollection = "twitter_sentiment"
+}
+
+class SVMExampleScala extends SparkApp.RDDProcessor  {
```

Comment:
Can you add Scaladoc here with brief summary and steps happening inside the class